modules/platforms/cpp/ignite/odbc/sql_environment.cpp

```diff
@@ -20,8 +20,15 @@
 
 namespace ignite {
 
-sql_connection *sql_environment::create_connection() {
-    sql_connection *connection;
+sql_environment::sql_environment()
+    : m_timer_thread(detail::thread_timer::start([&] (auto&& err) {
+        std::cout << "[SQL ENV CTOR] error: " << err.what() << std::endl;
```

Comment:
Use logger for logs like these. But it's probably is not needed - there is logging enabled when a new status record is added.